mule-extensions-api/src/main/java/org/mule/runtime/extension/api/runtime/operation/Result.java

```diff
@@ -30,7 +30,9 @@
  * @param <T> the generic type of the output value
  * @param <A> the generic type of the message attributes
  * @since 1.0
+ * @deprecated use {@link org.mule.sdk.api.runtime.operation.Result} instead.
  */
+@Deprecated
 public class Result<T, A> {
```

Comment:
leaving the Builder aside... can't this Result extend the SDK API Result?

Reply:
The Result class has a method copy() which returns a builder with the current state of the result. If I have Result extending from the SDK API Result, this copy method will be declared returning the other type of Builder. The option I see is to name this copy method different in the SDK Api Result. I don't think that option is valid.

Also, I could not make the extensions-api Result Builder access the fields in the SDK Api Result when I made them protected. I thought this would work but it seems that it does not.

Should I have approached it in a different way?